app/models/manageiq/providers/google/cloud_manager/provision/cloning.rb

```diff
@@ -0,0 +1,41 @@
+module ManageIQ::Providers::Google::CloudManager::Provision::Cloning
+  def do_clone_task_check(clone_task_ref)
+    source.with_provider_connection do |google|
+      instance = google.servers.get(clone_task_ref)
+      return true if instance.sshable?
+      return false
```

Comment:
When returning false the [poll_clone_complete](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/cloud_manager/provision/state_machine.rb#L20) method expects to get a status message back.

Even better, if you pass the instance ID back in `start_clone` below would this method be the same as the OpenStack [do_clone_task_check](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/openstack/cloud_manager/provision/cloning.rb#L3) method?